src/components/Dashboard/Dashboard.js

```diff
@@ -71,23 +71,25 @@ export const Dashboard = InjectAppServices(
       fetchData();
     }, [systemUsageSummary, dopplerSystemUsageApiClient]);
 
-    const handleCloseFirstStep = () => {
-      const updateData = async () => {
-        dispatchCompleteSteps({ type: COMPLETE_STEPS_ACTIONS.START_UPDATE });
-        const response = await dopplerSystemUsageApiClient.closeFirstSteps();
-        if (response.success) {
-          dispatchCompleteSteps({
-            type: COMPLETE_STEPS_ACTIONS.FINISH_UPDATE,
-          });
-        } else {
-          dispatchCompleteSteps({ type: COMPLETE_STEPS_ACTIONS.FAIL_UPDATE });
-        }
-      };
+    const { firstStepsClosedSince, completed } = firstStepsData;
 
-      updateData();
-    };
+    useEffect(() => {
+      if (!firstStepsClosedSince && completed) {
+        const updateData = async () => {
+          dispatchCompleteSteps({ type: COMPLETE_STEPS_ACTIONS.START_UPDATE });
+          const response = await dopplerSystemUsageApiClient.closeFirstSteps();
+          if (response.success) {
+            dispatchCompleteSteps({
+              type: COMPLETE_STEPS_ACTIONS.FINISH_UPDATE,
+            });
+          } else {
+            dispatchCompleteSteps({ type: COMPLETE_STEPS_ACTIONS.FAIL_UPDATE });
+          }
+        };
 
-    const { firstStepsClosedSince, completed } = firstStepsData;
+        updateData();
+      }
+    }, [completed, firstStepsClosedSince, dopplerSystemUsageApiClient]);
```

Comment:
If I understand it well, it simulates the user clicking the button.

Should we not simply ignore the flag without an additional request?

Reply:
I originally dismissed simply ignoring the flag because I wanted to show the animation of the transition between the first steps section and the quick actions section.

Reply:
Oh, I understand! you need the flag to know when to show the animation.

Thanks!